src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/CreateCcdService.java

```diff
@@ -35,35 +35,20 @@
 
     @Retryable
     public CaseDetails create(CaseData caseData, IdamTokens idamTokens) {
-        StartEventResponse startEventResponse = startEventCcdService.startCase(idamTokens.getAuthenticationService(),
-            idamTokens.getIdamOauth2Token(), "appealCreated");
-        CaseDataContent caseDataContent = CaseDataContent.builder()
-            .eventToken(startEventResponse.getToken())
-            .event(Event.builder()
-                .id(startEventResponse.getEventId())
-                .summary("GAPS2 Case")
-                .description("CaseLoader Case created")
-                .build())
-            .data(caseData)
-            .build();
-        return coreCaseDataApi.submitForCaseworker(
-            idamTokens.getIdamOauth2Token(),
-            idamTokens.getAuthenticationService(),
-            coreCaseDataProperties.getUserId(),
-            coreCaseDataProperties.getJurisdictionId(),
-            coreCaseDataProperties.getCaseTypeId(),
-            true,
-            caseDataContent);
+        return tryCreate(caseData, idamTokens);
```

Comment:
like this 👍

Reply:
Thanks @iguannago -- so just let me know if you would like anything else added/removed from this PR